test/library/standard/Path/victor-ludorum/testcases_isAbsPath.skipif

```diff
@@ -0,0 +1 @@
+CHPL_HOST_PLATFORM == linux32 , CHPL_HOST_PLATFORM == linux64 and CHPL_HOST_PLATFORM == darwin
```

Comment:
I don't know the details of the skipif file syntax, but I can't make sense of this one.

Reply:
@cassella  Actually I also don't know about the syntax . I have implemented it by considering linux32, linux64 and darwin environment . This is what I want to specify . Suggest me How can I write this ?? 
Thanks 😊 !!

Reply:
You'll be able to find that information by searching through the .skipifs used in the test directory.  I suggest looking up how to do a search to find all files with a specific suffix, and then looking through those for examples of chaining conditions

Reply:
Thanks @lydia-duncan  for your advice !! I have updated the **skipif** file . Kindly check it and suggest if any further modification is needed .